src/struct/Utils.ts

```diff
@@ -120,6 +121,11 @@ export class Utils {
 		context.textAlign = "left";
 		context.fillStyle = "#fff";
 		context.fillText(this.formatNumber(post.xp) + "/" + this.formatNumber(post.toLevel), 680, 145);
+
+		context.font = "13pt 'Poppins Light'";
+		context.textAlign = "left";
+		context.fillStyle = "#fff";
+		context.fillText("Voice Chat Time: " + String(Math.ceil(post.time / 1000 / 60) + "mins."), 524, 125);
```

Comment:
Showing the time as only minutes might be weird, can you format it like hh:mm:ss

Reply:
Well, I can merge the pr like this for now because I have something in my mind that fixes this line too, no need to change nothing here.